third_party/rust/src/libstd/BUILD

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright 2019 The Project Oak Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+load("@io_bazel_rules_rust//rust:rust.bzl", "rust_library")
+
+package(
+    licenses = ["notice"],
+)
```

Comment:
Do we need an `exports_files(["LICENSE-APACHE", "LICENSE-MIT"])` in a `BUILD` file somewhere?